pkgs/development/web/bun/default.nix

```diff
@@ -92,6 +92,7 @@ stdenvNoCC.mkDerivation rec {
       mit # bun core
       lgpl21Only # javascriptcore and webkit
     ];
+    mainProgram = "bun";
```

Comment:
Why to add mainProgram if pname is already `bun`?
Is this changing?

Reply:
Changed a while ago that relying on pname is deprecated. https://github.com/NixOS/nixpkgs/pull/246386

Reply:
Thanks for the reference.